third_party/Makefile.am

```diff
@@ -33,6 +33,7 @@ EXTRA_DIST                              = \
     nlbuild-autotools                     \
     openthread-test-driver                \
     NordicSemiconductor                   \
+    synopsys                              \
```

Comment:
Add note in the README to make clear that the current EMSK platform does not support a TRNG.